dspace/modules/additions/src/main/java/org/dspace/submit/step/ProjectStep.java

```diff
@@ -148,7 +142,11 @@ private int processProjectField(Context context, HttpServletRequest request, Ite
                 String funderAuthority = request.getParameter(MetadataField.formKey("rioxxterms", "funder", null) + "_authority");
                 try {
                     if (StringUtils.isBlank(funderAuthority)) {
-                        funderAuthority = new DSpace().getServiceManager().getServiceByName("defaultAuthorityCreator", DefaultAuthorityCreator.class).retrieveDefaultFunder(context).getId();
+                        FunderAuthorityValue defaultAuthority = new DSpace().getServiceManager().getServiceByName("defaultAuthorityCreator", DefaultAuthorityCreator.class).retrieveDefaultFunder(context);
+
+                        if(defaultAuthority!=null) {
+                            funderAuthority = defaultAuthority.getId();
+                        }
                     }
                     ProjectAuthorityValue project = projectService.createProject(context, value, funderAuthority);
```

Comment:
this code can't be reached if no default funder is configured, because the default project is retrieved earlier in the code. If any of the default values is null in the rioxx.cfg configuration, the project object retrieved will be null.